addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechsJsonParser.java

```diff
@@ -93,21 +94,28 @@ TechData parse(String categories, List<String> technologies, boolean createIcons
                         .map(
                                 path ->
                                         CompletableFuture.runAsync(
-                                                () ->
-                                                        parseJson(
-                                                                techData,
-                                                                getStringResource(path),
-                                                                createIcons),
+                                                () -> {
+                                                    if (LOGGER.isDebugEnabled()) {
```

Comment:
I'm not following why check debug and then log info, if debug is enabled the debug will be logged.

Reply:
It was an artifact from dev, when I decided I'd actually like it always to be in but I wanted to discuss it.